### DISCUSSION TITLE
Load all pluging instances and order of theriu when open a proyect....

### DISCUSSION THREAD

**Discussion by charlesramirezhof** [apohl79/audiogridder#627] [Help]
Hey when I do a load with many audiogrider tracks in a project. (Using LOGIC PRO on MAC AND WINDOWS AS SERVER)
For example, I usually use softube console1 in all LOGIC channels When making the final mix ... audiogrider does not load all the instances when I open the project ... I have to go to the connect all window because some instances are not reached to load.

Another problem that usually happens to me is that the order in which the plugings are loaded ... affects the track number that, for example, console one assigns to each channel. It would be nice to cascade the plugigs one by one in the order of the list.

A hug to all and thanks because it is one of the best tools and without a doubt the most revolutionary in years ...

**Comment by CaBuffalo** [comment]
In 1.2 Beta 3, you can save your own preset in AG server and load your
plugins with just one click.

Is this what you are looking for?

On Sat, May 15, 2021 at 12:24 PM charlesramirezhof ***@***.***>
wrote:

> Hey when I do a load with many audiogrider tracks in a project. (Using
> LOGIC PRO on MAC AND WINDOWS AS SERVER)
> For example, I usually use softube console1 in all LOGIC channels When
> making the final mix ... audiogrider does not load all the instances when I
> open the project ... I have to go to the connect all window because some
> instances are not reached to load.
>
> Another problem that usually happens to me is that the order in which the
> plugings are loaded ... affects the track number that, for example, console
> one assigns to each channel. It would be nice to cascade the plugigs one by
> one in the order of the list.
>
> A hug to all and thanks because it is one of the best tools and without a
> doubt the most revolutionary in years ...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/554>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FV6IGOLKRIJOI3KW3TN3C7LANCNFSM446FRTIA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by charlesramirezhof** [comment]
no I don't mean that ... i will try yo explain...(not easy)

when you load a project for example with console 1 ... this plug automatically assigns the track number to the mixer of the plug every time you load it into a track ... If you load it the first time it assigns it to track number 1 of the controller one mixer the second track that you load to track number 2 etc .....

but when loading the project again ... The green squares of the Pluging meter and the server ... in which you see the loaded plugs ... are loaded randomly, they do not start from top to bottom ... With which track numbers move ...

(The preset of each track are still well assigned but for example if the daw is track 2 and just loads that instance 5 of them, track two becomes 5 in the console one mixer) ...

**Comment by CaBuffalo** [comment]
I see. I don't have console 1. I believe it's a compatibility issue.

Let's wait and see what Andreas says about it.

On Sun, May 16, 2021 at 2:45 AM charlesramirezhof ***@***.***>
wrote:

> no I don't mean that ... i will try yo explain...(not easy)
>
> when you load a project for example with console 1 ... this plug
> automatically assigns the track number to the mixer of the plug every time
> you load it into a track ... If you load it the first time it assigns it to
> track number 1 of the controller one mixer the second track that you load
> to track number 2 etc .....
>
> but when loading the project again ... The green squares of the Pluging
> meter and the server ... in which you see the loaded plugs ... are loaded
> randomly, they do not start from top to bottom ... With which track numbers
> move ...
>
> (The preset of each track are still well assigned but for example if the
> daw is track 2 and just loads that instance 5 of them, track two becomes 5
> in the console one mixer) ...
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/issues/554#issuecomment-841793338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HIX6OWOSQK4PRNUTLTN6H4XANCNFSM446FRTIA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
There is no consistent way to access the track index of the DAW (I think only VST3 allows this, and it would require patching JUCE, as it is not passing this data). AG just indexes its instances by the load order.
